README.md

```diff
@@ -44,7 +44,9 @@ metadata:
   name: simplest
 spec:
   storage:
-    secret: minio-test
+    secret:
+      name: minio-test
+      type: s3
```

Comment:
What's the reasoning behind the storage type being a property of `secret` instead of `storage`?

Reply:
I had the same question. I copied the schema from the [`Loki Operator`](https://loki-operator.dev/docs/object_storage.md/) (for consistency) and I think is because the fields from the secret are different depending on the used storage.